packages/wallet/test/index.spec.ts

```diff
@@ -27,617 +27,659 @@ const isRunningInKarma = (): boolean => {
+    '.getAddress() should work'
+  )
+
+  t.throws(
+    function () {
       Wallet.fromPublicKey(pubKey).getPrivateKey()
-    }, /^Error: This is a public key only wallet$/)
-  })
-  // it('.toV3() should fail', function () {
+    },
+    (err: any) => err.message.includes('This is a public key only wallet'),
+    '.getPrivateKey() should fail'
+  )
+
+  // t.test('.toV3() should fail', function () {
   //   assert.throws(function () {
   //     Wallet.fromPublicKey(pubKey).toV3()
   //   }, /^Error: This is a public key only wallet$/)
   // })
+
+  t.end()
 })
 
-describe('.fromExtendedPrivateKey()', function () {
-  it('should work', function () {
-    const xprv =
-      'xprv9s21ZrQH143K4KqQx9Zrf1eN8EaPQVFxM2Ast8mdHn7GKiDWzNEyNdduJhWXToy8MpkGcKjxeFWd8oBSvsz4PCYamxR7TX49pSpp3bmHVAY'
-    assert.strictEqual(
-      Wallet.fromExtendedPrivateKey(xprv).getAddressString(),
-      '0xb800bf5435f67c7ee7d83c3a863269969a57c57c'
-    )
-  })
+tape('.fromExtendedPrivateKey()', (t) => {
+  const xprv =
+    'xprv9s21ZrQH143K4KqQx9Zrf1eN8EaPQVFxM2Ast8mdHn7GKiDWzNEyNdduJhWXToy8MpkGcKjxeFWd8oBSvsz4PCYamxR7TX49pSpp3bmHVAY'
+  t.deepEqual(
+    Wallet.fromExtendedPrivateKey(xprv).getAddressString(),
+    '0xb800bf5435f67c7ee7d83c3a863269969a57c57c'
+  )
+  t.end()
 })
 
-describe('.fromExtendedPublicKey()', function () {
-  it('should work', function () {
-    const xpub =
-      'xpub661MyMwAqRbcGout4B6s29b6gGQsowyoiF6UgXBEr7eFCWYfXuZDvRxP9zEh1Kwq3TLqDQMbkbaRpSnoC28oWvjLeshoQz1StZ9YHM1EpcJ'
-    assert.strictEqual(
-      Wallet.fromExtendedPublicKey(xpub).getAddressString(),
-      '0xb800bf5435f67c7ee7d83c3a863269969a57c57c'
-    )
-  })
+tape('.fromExtendedPublicKey()', (t) => {
+  const xpub =
+    'xpub661MyMwAqRbcGout4B6s29b6gGQsowyoiF6UgXBEr7eFCWYfXuZDvRxP9zEh1Kwq3TLqDQMbkbaRpSnoC28oWvjLeshoQz1StZ9YHM1EpcJ'
+  t.deepEqual(
+    Wallet.fromExtendedPublicKey(xpub).getAddressString(),
+    '0xb800bf5435f67c7ee7d83c3a863269969a57c57c'
+  )
+  t.end()
 })
 
-describe('.generate()', function () {
-  it('should generate an account', function () {
-    assert.strictEqual(Wallet.generate().getPrivateKey().length, 32)
-  })
-  it('should generate an account compatible with ICAP Direct', function () {
-    const max = BigInt('0x088f924eeceeda7fe92e1f5b0fffffffffffffff')
-    const wallet = Wallet.generate(true)
-    assert.strictEqual(wallet.getPrivateKey().length, 32)
-    const addr = bytesToHex(wallet.getAddress())
-    assert.strictEqual(BigInt('0x' + addr) <= max, true)
-  })
+tape('.generate()', (t) => {
+  t.equal(Wallet.generate().getPrivateKey().length, 32, 'should generate an account')
+  const max = BigInt('0x088f924eeceeda7fe92e1f5b0fffffffffffffff')
+  const wallet = Wallet.generate(true)
+  t.equal(
+    wallet.getPrivateKey().length,
+    32,
+    'should generate an account compatible with ICAP Direct'
+  )
+  const addr = bytesToHex(wallet.getAddress())
+  t.equal(BigInt('0x' + addr) <= max, true)
```

Comment:
```suggestion
  t.equal(BigInt('0x' + addr) <= max, true, 'should generate an account compatible with ICAP Direct')
```

Reply:
Could also remove it from the previous test, personally I think it makes more sense as the last message rather than the first (and that's what you seem to be doing in other tests)